### ISSUE TITLE
Unaccounted for [UserState]

### ISSUE THREAD

**Issue by bigfoott** [TwitchLib/TwitchLib.Client#101]
After connecting and sending messages I'm getting `Unaccounted for [UserState]: badge-info` in console. According to the source "This should never happen, unless Twitch changes their shit"

https://github.com/TwitchLib/TwitchLib.Client/blob/a1e90e3b87f4876a0293a4833cf05389a9010447/TwitchLib.Client.Models/UserState.cs#L88-L89

**Comment by Mahsaap** [comment]
Should be in dev branch

**Comment by JSpruill-LDS** [comment]
I am still getting this in vb.net
"Unaccounted for [UserState]: badge-info"
Using 3.0.4 compiled from source code.

**Comment by MrPashkovskiy** [comment]
I have sames problem. Does anyone have solution?

**Comment by JSpruill-LDS** [comment]
Yes, download and compile the dev branch will fix the issue. 
It would be nice to get a release with the patch though.

**Comment by JSpruill-LDS** [comment]
It was a frustrating one for me, it was causing my thread to die and the user was having to reconnect when their was lots of chat traffic. Prior to finding the dev code I wrote my own fix, then later found the dev code and closely matched. Also I found compiling the dll's in debug made proved to be very helpful in debugging other issues I was running into.

**Comment by swiftyspiffy** [comment]
Just to quickly reiterate from issue, the `Unaccounted for` message is a generic message that the parser spits out when it finds a property in the message from Twitch that isn't expected. It doesn't (shouldn't) cause any adverse impact on the application.
As the OP shows, it's just a Console.WriteLine in the default case (unless Console.WriteLine itself is causing problems?).

**Comment by JSpruill-LDS** [comment]
I agree with you that it shouldn't have been causing an issue, what I believe what was taking place was the rapid succession of errors would cause the thread to die.

we noticed it most commonly when viewers were having bit wars (trying to out donate one another) 
once I downloaded the source, and coded in the exception, it cesced to occur.

I won't be able to swap out the module till December, as this was a hobby and I have a deadline of Nov 18 to upgrade in parallel my systems at work with the HR upgrade. (its what pays the bills) but I will promptly test the first two weeks of Dec and provide feedback.

sorry for any delay this may cause anyone,
